src/braket/aws/aws_quantum_job.py

```diff
@@ -454,7 +463,7 @@ def result(
         poll_timeout_seconds: float = QuantumJob.DEFAULT_RESULTS_POLL_TIMEOUT,
         poll_interval_seconds: float = QuantumJob.DEFAULT_RESULTS_POLL_INTERVAL,
     ) -> dict[str, Any]:
-        """Retrieves the hybrid job result persisted using save_job_result() function.
+        """Retrieves the hybrid job result persisted using `save_job_result` function.
```

Comment:
nit: I believe "the `save_job_result` function" or just "`save_job_result`" sound more natural